platform/ios/PolyPodApp/PolyPod/en.lproj/Localizable.strings

```diff
@@ -36,8 +38,14 @@
 "button_confirm" = "Yes";
 "button_reject" = "No";
 "button_update_notification_close" = "OK";
-"auth_prompt_set_up" = "Set up authentification";
+
+"auth_prompt_set_up" = "Set up authentication";
+"re_auth_prompt_set_up" = "Re-authorize authentication";
```

Comment:
Why do we need separate strings for authentication setup from the settings? and for unlocking?

Reply:
it's feature request - FYI to be transparent to the users why they need to re-authorize